src/test/java/core/basesyntax/model/FruitTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertSame;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTest {
+    private static final String BANANA_NAME = "banana";
+    private static final String APPLE_NAME = "apple";
+    private static Fruit banana;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        banana = new Fruit("banana");
+    }
+
+    @Test
+    public void equalsTest_returnsTrue() {
+        assertEquals(banana, new Fruit("banana"));
```

Comment:
no need to test equals and hashCode methods

Reply:
deleted